lib/src/components/NxDrawer/NxDrawer.tsx

```diff
@@ -60,17 +63,19 @@ const NxDrawerHeader = (props: NxDrawerHeaderProps) => {
     children,
     ...attrs
   } = props;
-  const { closeDrawer } = useContext(NxDrawerContext);
+  const { closeDrawer, closeDisabled, closeBtnTooltip } = useContext(NxDrawerContext);
 
   const classes = classnames('nx-drawer-header', className);
 
   return (
     <header className={classes} role="none" {...attrs}>
-      <NxCloseButton className="nx-drawer-header__close-button"
-                     type="button"
-                     onClick={closeDrawer}>
-        Close
-      </NxCloseButton>
+      <NxTooltip title={closeBtnTooltip ? closeBtnTooltip : 'Close'}>
+        <NxCloseButton className={`nx-drawer-header__close-button ${closeDisabled ? 'disabled' : ''}`}
```

Comment:
Cleaner to use `classnames()` for this

Reply:
> Cleaner to use `classnames()` for this

Used `classnames()` here: https://github.com/sonatype/sonatype-react-shared-components/pull/1169/commits/4100080b3b0f6d792083e9961463daae9c37810d